ui/src/components/denali/RadioButton.js

```diff
@@ -108,6 +108,14 @@ const makeCssRadioButton = (props) => css`
         transition: ${!props.noanim ? 'all 0.2s ease-in' : undefined};
         width: 8px;
     }
+
+    /* Disabled checked button */
```

Comment:
Fixed a bug where a disabled checked radio button was only grey in the outer circle - the inner (checked) circle remained blue.